spec/system/admin_sign_in_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe 'Admin Sign in process', type: :system do
+  before do
+    driven_by(:rack_test)
```

Comment:
if you want you can put this line inside rails helper - it will run before every test that require rails_helper

Reply:
As suggested, I moved these lines to rails_helper and removed them from the tests